code/game/objects/items/devices/camera_bug.dm

```diff
@@ -43,3 +44,56 @@
 	. = ..()
 	integrated_console.network = list("ERT")
 
+/obj/item/wall_bug
+	name = "\improper small camera"
+	desc = "A camera with a sticky backside."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "wall_bug"
+	w_class = WEIGHT_CLASS_TINY
+	var/obj/machinery/camera/portable/camera
+	var/index = "REPORT THIS TO CODERS"
+
+/obj/item/wall_bug/Initialize(mapload, obj/item/camera_bug/the_bug)
+	. = ..()
+	link_to_camera(the_bug)
+	AddComponent(/datum/component/sticky, src)
```

Comment:
This can be removed too once the `I` arg from the sticky component `Initialize` is removed.